internal/cmd/kafka/command_cluster_create.go

```diff
@@ -89,9 +69,10 @@ func (c *clusterCommand) newCreateCommand(cfg *v1.Config) *cobra.Command {
 	pcmd.AddAvailabilityFlag(cmd)
 	pcmd.AddTypeFlag(cmd)
 	cmd.Flags().Int("cku", 0, `Number of Confluent Kafka Units (non-negative). Required for Kafka clusters of type "dedicated".`)
-	cmd.Flags().String("encryption-key", "", "Encryption Key ID (e.g. for Amazon Web Services, the Amazon Resource Name of the key).")
+	cmd.Flags().String("encryption-key", "", `Resource ID of the Cloud KMS key (GCP only).`)
```

Comment:
Spell out what KMS stands for?
```suggestion
	cmd.Flags().String("encryption-key", "", "Resource ID of the Cloud KMS key (GCP only).")
```

Reply:
`Resource ID of the Cloud Key Management Service key (GCP only).` reads worse than before imo, no strong opinion though.

Reply:
If this is something all of our customers will have heard of, let's keep it as is.